frontend/packages/console-app/src/__tests__/extension-checks/nav-items.spec.ts

```diff
@@ -0,0 +1,14 @@
+import { testedRegistry } from '../plugin-test-utils';
```

Comment:
This PR looks like more than updates to lodash imports?

Reply:
Hi Ben, this PR is a follow-up to un-merged #1708, above change comes from that PR.